e2e/ui/Identity Federation/error.spec.ts

```diff
@@ -0,0 +1,263 @@
+import { test as baseTest, expect, request } from '@playwright/test';
+import { ADMIN_PORTAL_PRODUCT, Portal, SSOPage } from 'e2e/support/fixtures';
+
+type MyFixtures = {
+  ssoPage: SSOPage;
+  samlFedPage: Portal;
+  oidcFedPage: Portal;
+};
+
+let oidcClientId;
+let oidcClientSecret;
+
+const FED_TENANT = 'acme.com';
+const FED_PRODUCT = '_jackson_admin_portal';
+
+const test = baseTest.extend<MyFixtures>({
+  ssoPage: async ({ page }, use) => {
+    const ssoPage = new SSOPage(page);
+    await use(ssoPage);
+    // Delete SSO Connections mapped to SAML federation
+    await ssoPage.deleteAllSSOConnections();
+  },
+  samlFedPage: async ({ page }, use) => {
+    const portal = new Portal(page);
+    // Create SAML Federated connection
+    await page.goto('/');
+    await page.getByRole('link', { name: 'Apps' }).click();
+    await page.waitForURL(/.*admin\/identity-federation$/);
+    await page.getByRole('button', { name: 'New App' }).click();
+    await page.waitForURL(/.*admin\/identity-federation\/new$/);
+    await page.getByPlaceholder('Your app').and(page.getByLabel('Name')).fill('SF-1');
+    await page.getByPlaceholder('example.com').and(page.getByLabel('Tenant')).fill(FED_TENANT);
+    await page.getByLabel('Product').fill(FED_PRODUCT);
+    await page.getByLabel('ACS URL').fill('https://invalid-url.com');
+    await page.getByLabel('Entity ID / Audience URI / Audience Restriction').fill('https://saml.boxyhq.com');
+    await page.getByRole('button', { name: 'Create App' }).click();
+    await page.waitForURL(/.*admin\/identity-federation\/.*\/edit$/);
+    await page.getByRole('link', { name: 'Back' }).click();
+    await page.waitForURL(/.*admin\/identity-federation$/);
+    await expect(page.getByRole('cell', { name: 'SF-1' })).toBeVisible();
+    // Add SAML connection via SAML Fed for Admin portal
+    await page.getByRole('link', { name: 'Single Sign-On' }).click();
+    await page.getByTestId('create-connection').click();
+    await page.getByLabel('Connection name (Optional)').fill('SSO-via-SAML-Fed');
+    await page
+      .getByPlaceholder('Paste the Metadata URL here')
+      .fill('http://localhost:5225/.well-known/idp-metadata');
+    await page.getByRole('button', { name: 'Save' }).click();
+    await expect(page.getByRole('cell', { name: 'SSO-via-SAML-Fed' })).toBeVisible();
+    await use(portal);
+    // Delete Saml Fed connection
+    await page.goto('/');
+    await page.getByRole('link', { name: 'Apps' }).click();
+    await page.waitForURL(/.*admin\/identity-federation$/);
+    await page.getByRole('cell', { name: 'Edit' }).getByRole('button').click();
+    await page.getByLabel('Card').getByRole('button', { name: 'Delete' }).click();
+    await page.getByTestId('confirm-delete').click();
```

Comment:
`deleteApp` function can be created to encapsulate this

Reply:
Will create a fixture class, same can be re-used across other spec files under Id Fed.